src/addons/editors/SimpleTextEditor.js

```diff
@@ -9,30 +9,15 @@
 var React                   = require('react');
 var keyboardHandlerMixin    = require('../../KeyboardHandlerMixin');
 var ExcelColumn             = require('../grids/ExcelColumn');
+var EditorBase              = require('./EditorBase');
 
-var SimpleTextEditor = React.createClass({
 
-  propTypes : {
-    onKeyDown : React.PropTypes.func.isRequired,
-    value : React.PropTypes.any.isRequired,
-    onBlur : React.PropTypes.func.isRequired,
-    column :  React.PropTypes.shape(ExcelColumn).isRequired
-  },
-
-  getValue(): any{
-    var updated = {};
-    updated[this.props.column.key] = this.refs.input.getDOMNode().value;
-    return updated;
-  },
-
-  getInputNode(): HTMLInputElement{
-    return this.getDOMNode();
-  },
+class SimpleTextEditor extends EditorBase {
 
   render(): ?ReactElement {
-    return (<input ref="input" type="text" onBlur={this.props.onBlur} className="form-control" defaultValue={this.props.value} onKeyDown={this.props.onKeyDown} />);
+    return (<input type="text" className="form-control" defaultValue={this.props.value} />);
   }
```

Comment:
@bakesteve, got bored in Ireland over Easter. Have simplified the editors. No longer have to worry about binding onKeyDown as this is taken care of in EditorContainer unless you want to override default behaviour